core/templates/pages/exploration-editor-page/editor-navigation/editor-navbar-breadcrumb.component.html

```diff
@@ -11,7 +11,7 @@
     <span ng-if="navbarTitle === ''" ng-click="editTitle()" class="oppia-untitled-exploration-text">
       Untitled Exploration
     </span>
-    <span ng-if="navbarTitle === null">
+    <span class="protractor-test-loading-fullpage" ng-if="navbarTitle === null">
```

Comment:
Added class `protractor-test-loading-fullpage` here as Loading message appears here only and so webdriver will correctly wait for the loading message to disappear in [link](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/waitFor.js#L67)
Earlier `protractor-test-loading-fullpage` was placed at a place which does represent loading message and so webdriver never waits for its invisibility at [link](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/waitFor.js#L67)

Reply:
Hi @SAEb-ai please use a separate waitFor method for this and use a new protractor class here if you specifically want to wait for this.

Reply:
Done